src/components/utils/validation.js

```diff
@@ -88,7 +88,7 @@ export function isVmNameValid (nameCandidate: string): boolean {
 }
 
 export function isDiskNameValid (nameCandidate: string): boolean {
-  return isVmNameValid(nameCandidate)
+  return /^[a-zA-Z0-9_\-\\.]{1,64}$/.test(nameCandidate)
```

Comment:
Why limiting to 64 chars while on webadmin and backend we now know that the limitation is for 255 chars? It's better to use the same limitation